### DISCUSSION TITLE
Over one CharacterAbilityModifyResistance will make resistance negative

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14402] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

if you have two or more talents that use CharacterAbilityModifyResistance to provide resistance your resistance will become negative. Not possible in vanilla since many damage resistance gained from talents are from unique afflictions

### Reproduction steps

1. download Baroverhaul a talent overhaul mod
2. wrench someone (2% blunt force trauma)
3. givetalent "Brick Wall" (10% resistance) and "Stormtrooper" (15% resistance)
4. wrench that character (4% blunt force trauma)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the next update. Closing discussion.

**Comment by A2ur3** [comment]
@Rokvach Tested in the latest version, and the issue is STILL not fixed. PLEASE address this 😭
